client/debian/apt-spacewalk/apt-spacewalk.spec

```diff
@@ -0,0 +1,150 @@
+%{!?__python2:%global __python2 /usr/bin/python2}
+
+%if %{undefined python2_version}
+%global python2_version %(%{__python2} -Esc "import sys; sys.stdout.write('{0.major}.{0.minor}'.format(sys.version_info))")
+%endif
+
+%if %{undefined python2_sitelib}
+%global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+%endif
+
+Name: apt-spacewalk
+Summary: Spacewalk plugin for Advanced Packaging Tool
+%if %{_vendor} == "debbuild"
+Packager: Spacewalk Project <spacewalk-devel@redhat.com>
```

Comment:
This probably needs to be adjusted to point to the Uyuni Project, with the appropriate mailing list contact email address.

Note, _do not_ remove the `Packager` tag under the debbuild vendor conditional, as it's required for Debian packages.

Reply:
Correct, but we need a short discussion about which address to use. @juliogonzalez - do you have an idea?

Reply:
I just requested uyuni-devel@opensuse.org, meant for cases such as this one and for development discussions, so you can use it for this PR.

I will send an email to uyuni-announce as soon as it's ready.